components/ghost_inspector_webhook/main.tf

```diff
@@ -0,0 +1,14 @@
+data "aws_ssm_parameter" "ghost_inspector_webhook" {
+  count = var.environment == "qa" ? 1 : 0
```

Comment:
I remember this `count` is a workaround way to do something but I need a refresher on what's happening here!

Reply:
My understanding is that this Terraform defines [`count`](https://www.terraform.io/docs/language/meta-arguments/count.html#basic-syntax) amount of resources. So it's our way of "turning it on" only for QA since on prod it would be `0`.

cc @DFurnes I could be completely misunderstanding!

Reply:
Ah that makes sense in this context to me!

Reply:
Yes, that's exactly it!